robot/wrapper.py

```diff
@@ -295,12 +329,15 @@ def wait_start(self) -> typing.Union[TEAM, None]:
         _logger.info("Robot started!\n")
         self.camera._start_thread()
 
-        return start_info["zone"]
+        return start_info
```

Comment:
I don't know what this does but it's changed from returning a element to returning the whole thing. Is that ok?